build/NuSpecs/Intellisense/Microsoft.Windows.AppLifecycle.Projection.xml

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<doc>
+  <assembly>
+    <name>Microsoft.Windows.AppLifecycle.Projection</name>
+  </assembly>
+  <members>
```

Comment:
members is empty, so no Intellisense support - should contain summaries for each public member, for example [Microsoft.UI.xml](https://dev.azure.com/microsoft/WinUI/_git/microsoft-ui-xaml-lift?path=%2Fpackaging%2Fintellisense%2FMicrosoft.UI.xml&_a=contents&version=GBmaster).

Note that while WinUI manually maintains xml doc files, it looks like VC++ 16.6 and later can now generate them from triple-slash comments:
https://devblogs.microsoft.com/cppblog/doxygen-and-xml-doc-comment-support/

Reply:
Cool. I see MRTCore's got one too -- `dev\MRTCore\packaging\Intellisense`. I'll delete that too (unlesss @huichen123 @axelandrejs squeak otherwise)

Reply:
please remove for MRT. Thanks.

Reply:
Per @Scottj1s correction these are needed. They should be fleshed out (need content, not just a trivial file as listed). I'll take a stab at it in a future PR per [DynamicDependencies: Flesh out Intellisense .xml with doccomments
#906](https://github.com/microsoft/ProjectReunion/issues/906)

There's some potential tooling to help. Once I figure it out we should be able to do likewise for AppLifecycle and MRTCore (<del>if</del> when :-) they contain doccomments)